.github/workflows/main.yml

```diff
@@ -0,0 +1,130 @@
+# This is a basic workflow to help you get started with Actions
+
+name: Flutter_CI
+
+# Controls when the workflow will run
+on:
+  # Triggers the workflow on push or pull request events but only for the "main" branch
+  push:
+    branches: [ "main" ]
```

Comment:
We will need to differentiate between the python and the flutter files. [This page talks about conditional work flows for github actions](https://docs.github.com/en/actions/using-workflows/events-that-trigger-workflows#running-your-workflow-based-on-the-head-or-base-branch-of-a-pull-request)

Reply:
Lets put both CI's in one file, I have mine almost done, we just need two separate folders for each part, the flutter and the python backend.

Reply:
I think we should keep the workflows separate. Single responsibility principle. Both can trigger on a PR to main branch. Also, it will be easier to maintain and read if the tools are separate. You should be able to create another `.yml` file under the `.github/workflows` and it should get picked up automatically.

Reply:
oh ok, thanks for letting me know. Is there a need for the conditional workload thing or will it automatically differentiate between python and the flow files?